python/ray/tests/test_client_builder.py

```diff
@@ -327,6 +327,10 @@ def has_client_deprecation_warn(warning: Warning, expected_replacement: str) ->
 @pytest.mark.filterwarnings(
     "default:Starting a connection through `ray.client` will be deprecated"
 )
+@pytest.mark.skipif(
```

Comment:
can you make a wrapper like "run_only_in_postmerge()"?

Reply:
Also @can-anyscale is there some sort of guarantee this env var won't change?

Reply:
I think it can only be guarantee through code reviews. But unless core change it, I don't think a random engineer will change it though.

Reply:
> can you make a wrapper like "run_only_in_postmerge()"?

I thought this would actually bring in additional dependency such as a `test_util.py` in....

Reply:
> > can you make a wrapper like "run_only_in_postmerge()"?
> 
> I thought this would actually bring in additional dependency such as a `test_util.py` of where we define this `run_only_in_postmerge()` func

Reply:
Okay. I don't think we will ever change the env. If there are times that this need to be changed, let us know! 

@rickyyx I think that's probably fine? We don't have that many tests anyway. I want to make sure we have corresponding issue (so we can ensure reason is always passed properly) whenever we mark it like this.

Reply:
> I want to make sure we have corresponding issue (so we can ensure reason is always passed properly) whenever we mark it like this.
that's a good point.

Reply:
actually - it's required to pass in reason with skipif using booleans.